test/cookbooks/macos_test/recipes/spotlight.rb

```diff
@@ -11,6 +11,10 @@
   end
 end
 
+execute 'test' do
+  command ['sudo', 'launchctl', 'unload', '-w', '/System/Library/LaunchDaemons/com.apple.metadata.mds.plist']
+end
+
```

Comment:
This would be a good place to functionally test the built-in `macosx_service` resource, too. See [this example](https://github.com/americanhanko/vsts-agent-macos-cookbook/blob/2cefa59c48eb4baf410bee7a741ec9d65c49a551/recipes/bootstrap.rb#L114-L118) from `vsts_agent_macos`. The `plist` key is undocumented.

2. I could be missing something, but is there any particular `sudo` used here? Chef runs as `root` by default unless we use the `user` property.